src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
FYI, the design doc for these APIs clearly spells out the reasoning for the implementations being internal and non-overrideable.